lambdas/replay-cimit-vcs/src/main/java/uk/gov/di/ipv/core/replaycimitvcs/ReplayCimitVcsHandler.java

```diff
@@ -92,9 +92,13 @@ private void handleBatch(List<ReplayItem> replayItems)
             VcStoreItem vcStoreItem =
                     this.verifiableCredentialService.getVcStoreItem(
                             item.getUserId().get("S"), item.getCredentialIssuer().get("S"));
-            SignedJWT vc = SignedJWT.parse(vcStoreItem.getCredential());
-            ciMitService.submitVC(vc, null, null);
-            submittedVcs.add(vc.serialize());
+            if (vcStoreItem.getCredential() != null) {
+                SignedJWT vc = SignedJWT.parse(vcStoreItem.getCredential());
+                ciMitService.submitVC(vc, null, null);
+                submittedVcs.add(vc.serialize());
+            } else {
+                LOGGER.warn("Null credential in VC");
```

Comment:
Can we include the user id/cri here, in case we need to take any further action on these?

Reply:
@Joe-Edwards-GDS are we alright to actually log that information? I assumed userId was off limits

Reply:
That's an interesting question - I think if it's not associated with any other info then it's not PII (it's basically a UUID), but it's probably safer to avoid if we don't need it (and we don't any more :)).